lab_3/main.py

```diff
@@ -20,7 +22,22 @@ def tokenize_by_sentence(text: str) -> tuple:
          (('_', 'h', 'e', '_'), ('_', 'i', 's', '_'), ('_', 'h', 'a', 'p', 'p', 'y', '_'))
          )
     """
-    pass
+
+    if not isinstance(text, str) or not text or not any(letter.isalpha() for letter in text):
+        return ()
+    end_punct = ['.', '?', '!']
```

Comment:
If the values of the array will not change, it is better to use `tuple`.

Reply:
ok done